src/sonic-host-services/scripts/caclmgrd

```diff
@@ -273,6 +274,38 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
         return allow_internal_docker_ip_cmds
 
+    def get_loopback_ip_from_loopback_keys(self, loopback_keys):
+
+        for key in loopback_keys:
+            if not _ip_prefix_in_key(key):
+                continue
+
+            iface_name, iface_cidr = key
+            if iface_name.startswith("Loopback3") and "/" in iface_cidr and "::" not in iface_cidr:
```

Comment:
Can you please simply this logic as in L#224 for getting IP addresses

Reply:
fixed